apps/web/lib/emails/templates/organizer-scheduled-email.ts

```diff
@@ -66,6 +69,8 @@ export default class OrganizerScheduledEmail {
       description: this.getTextBody(),
       duration: { minutes: dayjs(this.calEvent.endTime).diff(dayjs(this.calEvent.startTime), "minute") },
       organizer: { name: this.calEvent.organizer.name, email: this.calEvent.organizer.email },
+      ...(this.recurringEvent &&
+        this.recurringEvent.count && { recurrenceRule: new rrule(this.recurringEvent).toString() }),
```

Comment:
Event file sent on the email will have the recurrence rule to be created correctly on third-party calendars manually.